src/factories/ModuleFactory_v1.sol

```diff
@@ -12,6 +12,7 @@ import {IOrchestratorFactory_v1} from
 import {IInverterBeacon_v1} from "src/proxies/interfaces/IInverterBeacon_v1.sol";
 
 // Internal Dependencies
+
```

Comment:
<img src="https://github.com/InverterNetwork/contracts/assets/5999310/73816c13-edd6-4172-8e6d-90f0616cf352" data-canonical-src="https://github.com/InverterNetwork/contracts/assets/5999310/73816c13-edd6-4172-8e6d-90f0616cf352" width="250" />

Reply:
The linebreak here?
What triggers you here @marvinkruse ? xD